### ISSUE TITLE
Empty second rule is generated for static rules if other rule has fn value

### ISSUE THREAD

**Issue by vikramsunilpatil** [cssinjs/jss#668]
```
import * as React from 'react';
import injectSheet from 'react-jss';
import {Collapse, HamburgerIcon, Nav} from "../../../index";

const styles = {
  collapse: {
    display: "inline-flex",
    maxHeight: 999,
    '@media (max-width: 800px)': {
      display: "flex",
      flexDirection: "column",
      maxHeight: 0,
      overflow: "hidden",
      float: "none"
    }
  },
  show: {
    maxHeight: 999
  },
  navCollapse: {
    display: "inline-flex",
    float: "right",
    '@media (max-width: 800px)': {
      display: "flex",
      float: "none"
    }
  },
  icon: {
    display: "none",
    fontSize: 18,
    verticalAlign: "middle",
    color: props => props.navIconColor,
    '@media (max-width: 800px)': {
      display: "inline-block",
      position: "absolute",
      right: 0,
      top: "24px",
      paddingRight: "1rem"
    }
  }
};

class CollapsableNav extends React.PureComponent {
  static defaultProps = {
    navIconColor: "white"
  };

  render() {
    const {children,  classes} = this.props;
    return [
      <DiHamburgerIcon key="1" className={classes.icon}/>,
      <DiNav className={classes.navCollapse} key="2">
        <DiCollapse classes={classes}>
          {children}
        </DiCollapse>
      </DiNav>
    ]
  }
}

export default injectSheet(styles)(CollapsableNav);
```

Here props are being used in icon class to generate a dynamic stylesheet. The dynamic stylesheet has empty classes for the other classes. Can someone help with why those empty classes are being generated? How can i avoid those empty classes from being generated?
```
<style type="text/css" data-jss="" data-meta="CollapsableNav, Unthemed, Dynamic">
.CollapsableNav-icon-1-1-82 {
  color: black;
}
.CollapsableNav-collapse-1-1-83 {
}
.CollapsableNav-show-1-1-84 {
}
.CollapsableNav-navCollapse-1-1-85 {
}
</style>
```

**Comment by kof** [comment]
Can you please adda simplified example an publish it on codesandbox?

**Comment by vikramsunilpatil** [comment]
https://codesandbox.io/s/x3w304pw
In this example, you can see that empty class is generated for the header class in the dynamic stylesheet and nav class is passed a prop

**Comment by kof** [comment]
I confirm, the static rule without dynamic properties, should not generate a placeholder for a dynamic rule.

**Comment by abhishek199-dhn** [comment]
@kof having the same issue here, any workaround.?

**Comment by kof** [comment]
@abhishek199-dhn is it blocking you from anything?

**Comment by HenriBeck** [comment]
Fixed by cssinjs/react-jss#271
